ehitchhiking-ui/src/app/shared/components/active-trips-modal/active-trips-modal.component.ts

```diff
@@ -0,0 +1,85 @@
+import { Component, ElementRef, OnInit, ViewChild } from "@angular/core";
+import { LoaderSize } from "../../enums/pre-loader-sizes";
+import { MatDialogRef } from "@angular/material";
+import { ActiveTripsModalService } from "./active-trips-modal.service";
+import { Trip } from "../trips-modal/trips";
+import { ActiveTrip } from "../active-trip/active-trip";
+
+
+@Component({
+  selector: 'app-active-trips-modal',
+  templateUrl: './active-trips-modal.component.html',
+  styleUrls: ['./active-trips-modal.component.sass'],
+  providers: [ActiveTripsModalService],
+})
+export class ActiveTripsModalComponent implements OnInit {
+
+  limit = 5;
+  tripsArray = [];
+  tripsArrayLenght = 0;
+  loaderSize: LoaderSize = LoaderSize.Large;
+  loading = true;
+  scrollObserver: IntersectionObserver;
+  role = {roleField: 'role', isEnable: false}
+  selectedRole :number;
+  @ViewChild('sMarker', {static: true}) markerRef: ElementRef;
+  isShowTripInfo = false;
+  tripInfo: Trip;
+
+  roles = [
+    {value: 0, viewValue: 'Passenger'},
+    {value: 1, viewValue: 'Driver'},
+    {value: 2, viewValue: 'All'},
+  ];
+
+
+  constructor(public dialogRef: MatDialogRef<ActiveTripsModalComponent>, private tripService: ActiveTripsModalService) {
+    this.scrollObserver = new IntersectionObserver(this.onScroll.bind(this), {
+      threshold: 1,
+    });
+  }
+
+  onScroll(entries: IntersectionObserverEntry[]) {
+    for (let entry of entries) {
+      if (entry.isIntersecting) this.limit += 5;
```

Comment:
Why not 4? Pls use the same constant as in the "History" modal

Reply:
add brackets

Reply:
fixed